### PR TITLE
:bug: fix `nil` error when using `BootstrapOptions.Host`

### PR DESCRIPTION
**Pull request by ichekrygin**
Fixes: #271

### CODE REVIEW

**Comment by DirectXMan12** [comment]
looks good.  can you squash the commits together. (also /test won't work b/c we're not running PRs via prow yet :-/)

**Comment by ichekrygin** [comment]
@DirectXMan12 - rebase is done. 
It appears that "Linux" build failing (intermittent?). 
Is there a way to re-run the build?

**Comment by DirectXMan12** [comment]
I can do it.  Let me file a bug too.  I've got an issue to switch to prow now (#276) so that I don't have to do it.

**Comment by DirectXMan12** [comment]
filed a bug for the test flake too (#277)

**Comment by DirectXMan12** [comment]
/hold
wait a moment

File: pkg/webhook/bootstrap.go [hunk 1]
```diff
@@ -297,7 +297,7 @@ func (s *Server) validatingWHConfigs() (runtime.Object, error) {
 }
 
 func (s *Server) admissionWebhook(path string, wh *admission.Webhook) (*admissionregistration.Webhook, error) {
-	if wh.NamespaceSelector == nil && len(s.Service.Namespace) > 0 {
+	if wh.NamespaceSelector == nil && (s.Service == nil || s.Service.Namespace == "") {
```
[the remark this answers is not in the document]
Comment (ichekrygin) [reply pkg/webhook/bootstrap.go]: @DirectXMan12 this should be fixed, PTAL.

**Comment by mengqiy** [comment]
/lgtm
/hold cancel
Thanks for catching this and fixing this.

**Pull request merged**